contextid_test.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func getIdFromRequest(t *testing.T, server *httptest.Server) uint {
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done. Fixed in 25c8a2b061eb2c75af8ea98d0cb42faa6f4605d0